fitbenchmarking/fitting_benchmarking.py

```diff
@@ -154,45 +154,45 @@ def do_fitting_benchmark_one_problem(prob, minimizers, use_errors=True, count=0,
 
     # Get function definitions for the problem - one for each starting point
     function_defs = get_function_definitions(prob)
-    # search for lowest chi2
-    min_sum_err_sq = 1.e20
+
     # Loop over the different starting points
     for user_func in function_defs:
+        # search for lowest chi2
+        min_chi_sq = 1.e100  # used to store min chi-sq value found. initialised to large value
```

Comment:
instead of 1.e100 it would be even better to set to max floating point value

Reply:
Yes, I think I changed it by mistake. The maximum value should be 1e20.

Reply:
Hmm, this change was not present on my local branch.

Reply:
Fixed in commit 80decd4